### PR TITLE
Remote multinode scripts cleanup

### PR DESCRIPTION
**Pull request by pgarg66**
- Also added script to stop nodes

### CODE REVIEW

File: multinode-demo/stop_nodes.sh [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    set -ex; \
+    pkill -9 solana-; \
+    pkill -9 validator; \
+    pkill -9 leader; \
+  "
+  sleep 2
+  ((count++))
+done
+
+wait
```
Comment (mvines) [inline multinode-demo/stop_nodes.sh]: Nothing is going into the background, remove `wait`?

File: multinode-demo/stop_nodes.sh [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  echo -e "\\tUsage: $0 <IP Address array> <username> [path to ssh keys]\\n"
+  echo -e "\\t <IP Address array>: A bash script that exports an array of IP addresses, ip_addr_array. Elements of the array are public IP address of remote nodes."
+  echo -e "\\t <username>:         The username for logging into remote nodes."
+  exit 1
+}
+
+# Sample IP Address array file contents
+# ip_addr_array=(192.168.1.1 192.168.1.5 192.168.2.2)
+
+if [[ -z "$ip_addr_file" ]]; then
```
Comment (mvines) [inline multinode-demo/stop_nodes.sh]: I like using a one liner for this kind of thing: `[[ -n $ip_addr_file ]] || usage`

File: multinode-demo/stop_nodes.sh [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  usage
+fi
+
+if [[ -z "$remote_user" ]]; then
```
Comment (mvines) [inline multinode-demo/stop_nodes.sh]: No need to double quote in a `[[`

File: multinode-demo/stop_nodes.sh [hunk 4]
```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+ip_addr_file=$1
+remote_user=$2
+
+usage() {
+  echo -e "\\tUsage: $0 <IP Address array> <username> [path to ssh keys]\\n"
```
Comment (mvines) [inline multinode-demo/stop_nodes.sh]: How about this style: https://github.com/solana-labs/solana/blob/master/multinode-demo/gce_multinode.sh#L11-L31

**Comment by pgarg66** [comment]
I have added start and stop to the same script. I'll rename the start_nodes.sh to remote_nodes.sh in the next patch. Leaving it as is, so that it's easier to review the diffs.

**Comment by pgarg66** [comment]
@mvines , could you pls re-review it, accounting for my last comment above?

**Pull request merged**